docs/docs/development/docs.md

```diff
@@ -5,19 +5,115 @@
+`http://127.0.0.1:8081/`. It's a live-preview of the current version and does
+automatically refresh on file-changes.
 
-## Titles
+## Style guide
+
+### Titles
 
 When writing the documentation we are using title case in the
-[`Chicago Manual of Style` style](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style).
+[`Chicago Manual of Style`](https://en.wikipedia.org/wiki/Title_case#Chicago_Manual_of_Style).
 In addition personal names/ names of packages are written according to their
 documentation.
+
+### Cross-linking between pages
+
+When referencing other pages in the documentation, use relative links to the
+corresponding `.md` file. `MkDocs` will resolve the link.
+
+!!! warning
+
+    Do not use absolute links to other `.md` files in the same documentation,
+    those will break the links when the documentation is built.
+
+### Move documentation files
+
+When moving documentation files, make sure to add a redirect to the new
+location. This can be done by adding a line to the `mkdocs.yml` file:
+
+```yaml
+plugins:
+  - redirects:
+      redirect_maps:
+        'path/to/old/template.md': 'path/to/new/template.md' # Reason for the move
+```
+
+This ensures that we don't break any links to the old location, which are out
+of our control.
+
+### Tree structure
+
+The documentation is split into three main sections, each with its own
+subsections:
+
+<table>
+    <tr>
+        <th>Path</th>
+        <th>Name</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td><b>/user</b></td>
+        <td>User documentation</td>
+        <td>
+            For users of the Collaboration Manager. It contains information about how to use the platform and its features.
+            The content shall be written in a way that it is understandable for non-technical users.
+        </td>
+    </tr>
+    <tr>
+        <td>/user/project</td>
+        <td>Project documentation</td>
+        <td>
+            This section shall describe how to create, update and delete projects. It also contains information about the project settings and the project access management, as well as backup pipelines, tool models, diagram cache model sources and the model complexity badge.
+        </td>
+    </tr>
+    <tr>
+        <td>/user/sessions</td>
+        <td>Sessions documentation</td>
+        <td>
+            Describes tool indepdentend session features. It contains information how to create persistent and read-only sessions.
+            Tool specific features (e.g. TeamForCapella session tokens) don't belong here and should be documented in the tool section.
+        </td>
+    </tr>
+    <tr>
+        <td>/user/tools</td>
+        <td>Tools documentation</td>
+        <td>
+            Tool specific content belongs here. Examples are Capella related topics, for example the TeamForCapella and Git workflows.
+            Also Jupyter, Papyrus and `pure::variants` configurations shall be documented here. <br \> <br \>
```

Comment:
Jupyter, Papyrus and `pure::variants` configurations should also be documented here.